single_cell/workflows/qc/scripts/single_cell_qc_plots.py

```diff
@@ -326,7 +343,7 @@ def load_qc_data(sample_id, annotation_metrics, hmmcopy_reads, hmmcopy_segs, hmm
     alignment_metrics, gc_metrics
 ):
 
-    results_tables_new = scgenome.db.qc.get_qc_data_from_filenames( [annotation_metrics], [hmmcopy_reads], 
+    results_tables_new = scgenome.db.qc_from_files.get_qc_data_from_filenames( [annotation_metrics], [hmmcopy_reads], 
```

Comment:
lets indent. 

make it look like this

scgenome.db.qc_from_files.get_qc_data_from_filenames(
    [annotation_metrics], [hmmcopy_reads] ...